stockflux-securities-master/src/components/ErrorDisplayer.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./ErrorDisplayer.css";
+
+const ErrorDisplayer = ({ errorMessage }) => {
```

Comment:
Can we rename this component to `ErrorMessage`?

Reply:
done